client/src/App.css

```diff
@@ -1,3 +1,42 @@
+:root {
+    /* These are all the defaults for CoTripper. Below is an example of usage:
+        background-color: var(--yellow);
+    */
+
+    /* Default color palette variables */
+    --yellow: #FFCB06;
+    --pink: #FB8370;
+    --red: #D1495B;
+    --purple: #6A4C93;
+    --black: #252525;
+    --white: #FAFAFA;
+    /* Default font */
+    --default-font: 'Reem Kufi', sans-serif;
+    /* Images */
+    --comment-black: url("../../assets/images/comment_black.png");
+    --edit-button: url("../../assets/images/edit_button.png");
+    --global-nav-dm: url("../../assets/images/global-nav_dm_icon.png");
+    --global-nav-logo: url("../../assets/images/global-nav_logo_icon.png");
+    --global-nav-search: url("../../assets/images/global-nav_search_icon.png");
+    --heart-pink: url("../../assets/images/heart_pink.png");
+    --heart-white: url("../../assets/images/heart_white.png");
+    --heart-black: url("../../assets/images/heart_black.png");
+    --home-banner: url("../../assets/images/home_banner.png");
+    --login-logo: url("../../assets/images/login_logo.png");
+    --nav-arrow-purple: url("../../assets/images/nav_arrow_purple.png");
+    --nav-arrow-yellow: url("../../assets/images/nav_arrow_yellow.png");
+    --social-facebook: url("../../assets/images/social_facebook.png");
+    --social-instagram: url("../../assets/images/social_instagram.png");
+    --social-pinterest: url("../../assets/images/social_pinterest.png");
+    --slash-background: url("../../assets/images/splash_background.png");
+    --text-box-search: url("../../assets/images/text_box_search.png");
+    /* Font Weights */
+    --paragraph-weight: 14px;
+    --banner-weight: 80px;
+    --link-weight: 24px;
+    --title-weight: 36px;
```

Comment:
These need to be in units of 100 (i.e. 100, 200, 300, etc up to 900) - like we talked about in person

Reply:
Okay. This is fixed and should be ready. There's a pull req for it currently.

Reply:
@ZakkMan Can we get this merged ASAP so that everyone can get to using them? Thanks!